\hypertarget{class_n_e_a_t___py_genetics_1_1_n_e_a_t_1_1_error_handling_1_1_exceptions_1_1_network_timeout_ex09d61843cecde07531bc3d1c6f238646}{}\section{N\+E\+A\+T\+\_\+\+Py\+Genetics.\+N\+E\+A\+T.\+Error\+Handling.\+Exceptions.\+Network\+Timeout\+Exception.\+Network\+Timeout\+Exception Class Reference}
\label{class_n_e_a_t___py_genetics_1_1_n_e_a_t_1_1_error_handling_1_1_exceptions_1_1_network_timeout_ex09d61843cecde07531bc3d1c6f238646}\index{N\+E\+A\+T\+\_\+\+Py\+Genetics.\+N\+E\+A\+T.\+Error\+Handling.\+Exceptions.\+Network\+Timeout\+Exception.\+Network\+Timeout\+Exception@{N\+E\+A\+T\+\_\+\+Py\+Genetics.\+N\+E\+A\+T.\+Error\+Handling.\+Exceptions.\+Network\+Timeout\+Exception.\+Network\+Timeout\+Exception}}
Inheritance diagram for N\+E\+A\+T\+\_\+\+Py\+Genetics.\+N\+E\+A\+T.\+Error\+Handling.\+Exceptions.\+Network\+Timeout\+Exception.\+Network\+Timeout\+Exception\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=1.848185cm]{class_n_e_a_t___py_genetics_1_1_n_e_a_t_1_1_error_handling_1_1_exceptions_1_1_network_timeout_ex09d61843cecde07531bc3d1c6f238646}
\end{center}
\end{figure}
\subsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
def {\bfseries \+\_\+\+\_\+init\+\_\+\+\_\+} (self, message=\char`\"{}\char`\"{}, errors=None)\hypertarget{class_n_e_a_t___py_genetics_1_1_n_e_a_t_1_1_error_handling_1_1_exceptions_1_1_network_timeout_ex09d61843cecde07531bc3d1c6f238646_ac78b36042b085235de601a15a68ce98a}{}\label{class_n_e_a_t___py_genetics_1_1_n_e_a_t_1_1_error_handling_1_1_exceptions_1_1_network_timeout_ex09d61843cecde07531bc3d1c6f238646_ac78b36042b085235de601a15a68ce98a}

\end{DoxyCompactItemize}


\subsection{Detailed Description}


Definition at line 1 of file Network\+Timeout\+Exception.\+py.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
N\+E\+A\+T/\+Error\+Handling/\+Exceptions/Network\+Timeout\+Exception.\+py\end{DoxyCompactItemize}
